homeassistant/components/yolink/sensor.py

```diff
@@ -157,6 +168,41 @@ def cvt_battery(val: int | None) -> int | None:
         entity_registry_enabled_default=False,
         should_update_entity=lambda value: value is not None,
     ),
+    YoLinkSensorEntityDescription(
+        key="state",
+        device_class=SensorDeviceClass.ENUM,
+        name="State",
+        icon="mdi:flash",
+        options=["normal", "alert", "off"],
+        exists_fn=lambda device: device.device_type in ATTR_DEVICE_POWER_FAILURE_ALARM,
+    ),
+    YoLinkSensorEntityDescription(
+        key="mute",
+        device_class=SensorDeviceClass.ENUM,
+        name="Mute",
+        icon="mdi:volume-mute",
+        options=["yes", "no"],
+        exists_fn=lambda device: device.device_type in ATTR_DEVICE_POWER_FAILURE_ALARM,
+        value=lambda value: "yes" if value is True else "no",
```

Comment:
Since this is not a control but a sensor, should the states be "muted" and "unmuted"?

Reply:
There is a physical mute button on the device, This entity is used to show whether the device is in silent mode